src/tokens/fontTokens.stories.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+  import { font as allFonts } from '../../tokens/css'
+  import FontTokenSwatchGroup from '../storyHelpers/FontTokenSwatchGroup.svelte'
+
+  const { macos, monospace, windows, primary, secondary, mobile, components } =
+    allFonts
+</script>
+
+<Meta title="Font Tokens" />
+
+<Story name="All Fonts">
+  <FontTokenSwatchGroup tokens={allFonts} />
+</Story>
+
+<Story name="macOS">
+  <FontTokenSwatchGroup tokens={macos} />
```

Comment:
This one doesn't seem to work for me on the deployed storybook - maybe 'cause I don't have a mac?

Reply:
Can't hurt for the storybook - it's a bit weird to have entries for things which are going to display differently when you actually use them